bigtable-dataflow-parent/bigtable-hbase-beam/pom.xml

```diff
@@ -0,0 +1,188 @@
+<!--
+Copyright 2017 Google Inc. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.google.cloud.bigtable</groupId>
+        <artifactId>bigtable-dataflow-parent</artifactId>
+        <version>1.0.0-pre3-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>bigtable-hbase-beam</artifactId>
+    <packaging>jar</packaging>
+    <name>${project.groupId}:${project.artifactId}</name>
+    <description>
+       This project contains artifacts that provide Cloud Bigtable client readers and writers in Google Cloud Dataflow.
+    </description>
+
+    <properties>
+        <beam.version>2.0.0</beam.version>
+        <commons-lang.version>2.6</commons-lang.version>
+        <google-auto-service.version>1.0-rc2</google-auto-service.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.beam</groupId>
+            <artifactId>beam-sdks-java-core</artifactId>
+            <version>${beam.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.auto.service</groupId>
+            <artifactId>auto-service</artifactId>
+            <version>${google-auto-service.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.cloud.bigtable</groupId>
+            <artifactId>bigtable-hbase-1.x-shaded</artifactId>
```

Comment:
Looking at the beam mailing list, 2.1 is at RC3 with no blocking issues and will land in the next couple of days.  Maybe we should jump to that version and avoid introducing shaded artifacts?
http://mail-archives.apache.org/mod_mbox/beam-dev/201708.mbox/%3C15774BA3-1E98-4A0B-BD80-6E6EC14F6941%40apache.org%3E